### DISCUSSION TITLE
How does this compare to the new official MS OpenAI Go SDK

### DISCUSSION THREAD

**Discussion by PylotLight** [sashabaranov/go-openai#768] [General]
Hey,

Was curious as to how you see these compared and whether they support different features and have different focuses given how long go-openai has been around and that it's still fully featured and updated regularly vs the new official client from MS. Sure they overlap but where would they not overlap, and how do we feel about these going forward?

Please forgive if not appropriate.

https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/ai/azopenai

**Comment by arykalin** [comment]
As I know azure openai DSK was publicly release several months ago. So main thing that sasha's repo is older. Though I'd like to hear those who try it.

**Comment by YugoCode** [comment]
There is now also an official Golang SDK from OpenAI:

https://github.com/openai/openai-go
